platform/commonUI/general/src/ui/TreeView.js

```diff
@@ -60,12 +59,10 @@ define([
 
     TreeView.prototype.loadComposition = function () {
         var self = this,
-            domainObject = this.activeObject;
-
```

Comment:
@aayusharora - lets keep the empty lines between variables and functions, for readability